flask_fluentd.py

```diff
@@ -36,8 +36,10 @@ def send_events(self, exception):
         pumping = True
         while pumping:
             try:
-                tag, evt = self.queue.get()
+                tag, evt = self.queue.get_nowait()
                 self._sender.emit(tag, evt)
                 self.queue.task_done()
             except Empty:
                 pumping = False
+
+        return exception
```

Comment:
It's the exception that was generated in the request (if one was) or the response object (if it happened cleanly). It's a flask thing.